ckanext/who_afro/helpers.py

```diff
@@ -201,3 +203,12 @@ def get_license(license_id):
     else:
         return {}
 
+
+def get_package_stats(package_id):
```

Comment:
This feels to me like lasagne code for no reason? Surely we should just call the upstream package_stats action directly in our templates rather than wrapping them like this?

Reply:
I don't think that's a good idea because it the information is no longer local. This also allows us to customize the data to be displayed in the template, keeping templates lightweight.

Also we shouldn't call the upstream `package_stats` directly because CKAN messes up with imports. See the docs of `get_action` at https://docs.ckan.org/en/latest/extensions/plugins-toolkit.html#ckan.plugins.toolkit.ckan.plugins.toolkit.get_action.

To quote:
> You should always use get_action() instead of importing an action function directly, because [IActions](https://docs.ckan.org/en/latest/extensions/plugin-interfaces.html#ckan.plugins.interfaces.IActions) plugins can override action functions, causing get_action() to return a plugin-provided function instead of the default one.

Reply:
But with the solution I am proposing there is no importing of the action at all. Because `ckanext-google-analytics` will have registered the two actions `package_stats` and `resource_stats` we would completely remove all the changes you have made to `helpers.py` and to `plugin.py` and the template changes, e.g. `resource_item.html` would become:
`<p class="resource-detail file-size">{{h.localised_filesize(res.size)}} modified {{h.time_ago_from_timestamp(res.last_modified) }} &middot; Downloads: {{ h.resource_stats(res.id) }}</p>`.

Of course, if we take this approach we probably want to rename the actions in `ckanext-google-analytics` to `get_*` but that's a minor change to make.

Reply:
Done!